### ISSUE TITLE
Issue with Champion Scans

### ISSUE THREAD

**Issue by KucingCyber** [riderkick/FMD#1593]
the link for Champion Scans doesn't works, it seems that they update their link,
e.g. from [https://reader.championscans.com/series/lv999-no-murabito/](url)
to [https://reader.championscans.com/r/series/lv999-no-murabito/](url)
and when i change the url to the new link it show nothing.
can anybody help?
sorry for my bad english.

**Comment by Tmp341** [comment]
Yeah. There was a problem with Champion Scans. I haven't checked the problem. We just need an update.

**Comment by arbiter9605** [comment]
They had to move to a new server and seems like reader changed that they use so URL's are some what different. They posted on site its all been uploaded now.
